extensions/cornerstone/src/hps/frameView.ts

```diff
@@ -9,7 +9,7 @@ const frameView: Types.HangingProtocol.Protocol = {
   toolGroupIds: ['default'],
   protocolMatchingRules: [],
   displaySetSelectors: {
-    defaultDisplaySetId: {
+    activeDisplaySet: {
```

Comment:
Why are you renaming this to active display set?  There is a difference between the default display set and the active one.

Reply:
It is being renamed so that if the HP is reapplied it will use whatever display set was swapped in. For instance take this scenario.., 
1. start with the frame view HP 
2. double click to one-up
3. drag in a different series
4. reapply (i.e. choose) the frame view HP
5. The display set dragged in from 3 should display. Without this, the original display set from 1 will show.

Reply:
Yes, I understand now what it is doing - I was confused with the matchedDisplaySetsIndex value, which is causing an issue.